pkg/controller/expcertificates/issuing/secret.go

```diff
@@ -157,36 +127,53 @@ func (s *secretsManager) setValues(crt *cmapi.Certificate, secret *corev1.Secret
 		secret.Data = make(map[string][]byte)
 	}
 
-	// Handle the experimental PKCS12 support
-	if s.experimentalIssuePKCS12 {
-		// Only write a new PKCS12 file if any of the private key/certificate/CA data has
-		// actually changed.
-		if data.sk != nil && data.cert != nil &&
-			(!bytes.Equal(secret.Data[corev1.TLSPrivateKeyKey], data.sk) ||
-				!bytes.Equal(secret.Data[corev1.TLSCertKey], data.cert) ||
-				!bytes.Equal(secret.Data[cmmeta.TLSCAKey], data.ca)) {
-			keystoreData, err := encodePKCS12Keystore(s.experimentalPKCS12KeystorePassword, data.sk, data.cert, data.ca)
+	// Only write a new PKCS12/JKS file if any of the private key/certificate/CA
+	// data has actually changed.
+	if data.sk != nil && data.cert != nil &&
```

Comment:
not sure why `pk` got renamed to `sk` when the issuing controller was written but `sk` is the private key here 🙄

Reply:
`pk` = public key, `sk` = secret (private) key